commit 2df7ddf1561ae70dc8c59796eb2f35dd8b0a58f4
Author: Denis Arnaud <denis.arnaud_fedora@m4x.org>
Date:   Tue May 18 00:05:03 2021 +0200

    Fix build when SIGSTKSZ is no longer a constant
    
    In the latest glibc versions SIGSTKSZ is not a constant any more, which
    broke building the tests with it.
    
    Work around this by hard-coding a typical value for it.
    
    closes #886.

diff --git a/tests/catch.hpp b/tests/catch.hpp
index 014df2c5..85067813 100644
--- a/tests/catch.hpp
+++ b/tests/catch.hpp
@@ -6489,7 +6489,7 @@ namespace Catch {
         static bool isSet;
         static struct sigaction oldSigActions [sizeof(signalDefs)/sizeof(SignalDefs)];
         static stack_t oldSigStack;
-        static char altStackMem[SIGSTKSZ];
+        static char altStackMem[8192];
 
         static void handleSignal( int sig ) {
             std::string name = "<unknown signal>";
@@ -6540,7 +6540,7 @@ namespace Catch {
     bool FatalConditionHandler::isSet = false;
     struct sigaction FatalConditionHandler::oldSigActions[sizeof(signalDefs)/sizeof(SignalDefs)] = {};
     stack_t FatalConditionHandler::oldSigStack = {};
-    char FatalConditionHandler::altStackMem[SIGSTKSZ] = {};
+    char FatalConditionHandler::altStackMem[8192] = {};
 
 } // namespace Catch
 
